src/lib/crypto/builtin/sha2/sha256.c

```diff
@@ -211,7 +211,7 @@ k5_sha256_update(SHA256_CTX *m, const void *v, size_t len)
 #if !defined(WORDS_BIGENDIAN) || defined(_CRAY)
 	    int i;
 	    uint32_t current[16];
-	    struct x32 *u = (struct x32*)m->save;
+	    struct x32 *u = (struct x32*)(void*)m->save;
```

Comment:
Building on an Intel platform, WORDS_BIGENDIAN would be false, so only the first branch gets compiled.